src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -710,5 +693,141 @@ public void SarifTransformerTests_ToCurrent_OneRunWithNotificationsButNoInvocati
+                              ""frames"": [
+                                {
+                                  ""message"": ""Exception thrown"",
+                                  ""module"": ""RuleLibrary"",
+                                  ""threadId"": 52,
+                                  ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Evaluate"",
+                                  ""address"": 10092852
+                                },
+                                {
+                                  ""module"": ""ExecutionEngine"",
+                                  ""threadId"": 52,
+                                  ""fullyQualifiedLogicalName"": ""ExecutionEngine.Engine.EvaluateRule"",
+                                  ""address"": 10073356,
+                                  ""offset"": 10475
+                                }
+                              ]
+                            },
+                            ""innerExceptions"": [
+                              {
+                                ""kind"": ""System.ArgumentException"",
+                                ""message"": ""length is < 0""
+                              }
+                            ]
+                          }
+                        }
+                      ],
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            string v2LogText = JsonConvert.SerializeObject(v2Log, SarifTransformerUtilities.JsonSettingsV2);
+            string v2LogExpectedText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""invocations"": [
+        {
+          ""toolNotifications"": [
+            {
+              ""id"": ""CTN0001"",
+              ""message"": {
+                ""text"": ""Run started.""
+              },
+              ""level"": ""note"",
+              ""exception"": {
+                ""kind"": ""ExecutionEngine.RuleFailureException"",
+                ""message"": ""Unhandled exception during rule evaluation."",
+                ""stack"": {
+                  ""message"": {
+                    ""text"": ""This is the stack messasge.""
+                  },
+                  ""frames"": [
+                    {
+                      ""module"": ""RuleLibrary"",
+                      ""threadId"": 52,
+                      ""address"": 10092852,
+                      ""properties"": {
+                        ""sarifv1/message"": ""Exception thrown""
```

Comment:
You noticed that the v2 `stackFrame` object doesn't have a `message` property. But it does have a `location` property, which has a `message`, so write:
```
{
  "module": "RuleLibrary",
  ...
  "location": {
    "message": {
      "text": "Exception thrown."
    }
  }
}
```

Reply:
Ah ok. I haven't done location yet (next batch).

Reply:
Ok. How are we going to remember to fix this when you do the `location` PR?

Reply:
A sticky note :-)

Reply:
Yeah I have a method that has worked so far :)